src/test/mir-opt/move_slice.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+pub fn test(a: &[u64; 8]) -> [u64; 8] {
```

Comment:
This file's name has "slice" in it but these are arrays being copied.
(`move_chain` seems more relevant, the type itself doesn't really matter)